paparazzi-gradle-plugin/src/test/projects/apply-out-of-order/src/test/java/app/cash/paparazzi/plugin/test/VerifyTest.kt

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2020 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package app.cash.paparazzi.plugin.test
+
+import android.content.Context
+import android.widget.LinearLayout
+import app.cash.paparazzi.Paparazzi
+import org.junit.Rule
+import org.junit.Test
+
+class VerifyTest {
+  @get:Rule
+  val paparazzi = Paparazzi()
+
+  @Test
+  fun verify() {
+    paparazzi.snapshot(LinearLayout(paparazzi.context))
+  }
+}
```

Comment:
this test file can be removed.  `missingPlugin` can be used as a template for this test.

Reply:
It needs to check that it actually does a snapshot when the plugins are appied the other way around, missingPlugin only verifies a failure.

Reply:
not exactly.  the goal is to check that plugin ordering doesn't matter, which can be verified without continuing on to run snapshots, i.e., the prepareResources task run is sufficient to assert that.